react_practice/react-apps/waitlist-page-v1/src/CTAForm.js

```diff
@@ -1,5 +1,6 @@
 import React, { useState } from "react";
-import PrimaryButton from "./PrimaryBtn";
+import PrimaryBtn from "./PrimaryBtn";
```

Comment:
As a rule of thumb, I find it helpful to name the component the same as the file. This way when you're scrolled down in your editor and looking at the code below, its easy for you to find the definition of the `PrimaryBtn` without needing to come back to the `import` to find what file its from. Not the end of the world if its named different, but where you can, its nice to have them match as an organizational tool.